GeoQuiz/app/src/main/java/com/bignerdranch/android/geoquiz/MainActivity.kt

```diff
@@ -0,0 +1,85 @@
+package com.bignerdranch.android.geoquiz
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import android.widget.ImageButton
+import android.widget.TextView
+import android.widget.Toast
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var trueButton: Button
+    private lateinit var falseButton: Button
+    private lateinit var nextButton: ImageButton
+    private lateinit var questionTextView: TextView
+    private lateinit var prevButton: ImageButton
```

Comment:
try not to use abbreviations that can mean multiple things as it won't be intuitive for someone seeing it for the first time, prev could stand for multiple things like previous, preview etc.